samples/basic/blinky/prj.conf

```diff
@@ -1 +1,7 @@
-CONFIG_GPIO=y
+# CONFIG_GPIO=y
```

Comment:
`CONFIG_GPIO` should be enabled, only `CONFIG_GPIO_NRFX` should be explicitly disabled here.